ignopeverell/grin
# Tests for input output commitments

This PR includes the following - 

* fixup `grin_wallet` tests 
  * add `rustc-serialize` to Cargo.toml in wallet dir
  * zero'd out fingerprint was causing test to fail
* add new test to verify equality of input and output commitments when spending a coin

That last point was not immediately obvious to me. With hindsight it makes more sense but it is still far from intuitive that the commitments are actually _the same_ without writing it down here as a test.
